packages/@cdktf/hcl2cdk/lib/generation.ts

```diff
@@ -59,6 +62,75 @@ function getReference(graph: DirectedGraph, id: string) {
   }
 }
 
+function destructureAst(ast: t.Expression): string[] | undefined {
```

Comment:
What does this function do? Create a path of an expression? A small comment with an example would really help.

Reply:
Added a comment